envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/resource/listeners/filters/RBACFilterPermissions.kt

```diff
@@ -44,18 +50,45 @@ class RBACFilterPermissions {
     }
 
     private fun createPathMatcher(path: String, matchingType: PathMatchingType): PathMatcher {
-        val matcher = when (matchingType) {
-            PathMatchingType.PATH -> StringMatcher.newBuilder().setExact(path).build()
-            PathMatchingType.PATH_PREFIX -> StringMatcher.newBuilder().setPrefix(path).build()
+        val matcher = when (path.contains(pathParamRegex)) {
```

Comment:
Wouldn't be easier to create third path matching type?
Right now user can define in tycho.yaml:
* `path`
* `pathPrefix`

We can add third option:
* `pathRegex`

Then we can simplify the logic (we don't have to remove `.*` from the end of the path)

Reply:
I assumed we just wanted to add params in paths and pathPrefixes... That's why I didn't create another type (regex). It would be the easiest way but we will allow all regex characters in segments also. Moreover users will have to rember to use `pathRegex` instead of `path` or `pathPrefix` if path contains parameters